### PR TITLE
Adds react native debugger command and instructions

### PR DESCRIPTION
**Pull request by l2succes**
screen shot 2017-01-23 at 1 00 01 pm

### CODE REVIEW

**Comment by alloy** [comment]
Awesome :+1:

File: package.json [hunk 1]
```diff
@@ -73,6 +74,7 @@
     "jest": "^18.1.0",
     "jest-cli": "^17.0.3",
     "jest-react-native": "^17.0.3",
+    "react-native-debugger-open": "^0.3.2",
```
Comment (orta) [inline package.json]: I see no lock file changes for this?

Comment (l2succes) [inline package.json]: Yeah, actually I'll remove it for now as I included a launch command in `package.json` instead.

**Comment by sarahscott** [comment]
🔥

**Pull request merged**